Resources/Prototypes/Entities/Structures/Lighting/base_lighting.yml

```diff
@@ -308,7 +327,7 @@
   - type: AmbientSound
     sound:
       path: /Audio/Ambience/Objects/alarm.ogg
-    volume: -12
```

Comment:
See above comment

Reply:
Because I was told the current blue light is incredibly bright to the point of being almost useless inside the station, and I was forwarded these and told to PR these because Glacier uses the blue lights a lot.

Reply:
You introduced a new prototype, which is fine, but you made changes to unrelated prototypes

Reply:
Honestly I was forwarded these by Peptide, so I don't actually know what that's about